setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import find_packages
+import os
+from setuptools import setup, Command
```

Comment:
Too clumpy imports. You are importing from `setuptools` in two different lines. Surely, it should be done in one line.

So your first two lines (note the order) should be:

```python
import os
from setuptools import setup, find_packages, Command
```